### PR TITLE
add documentation for our image version strategy

### LINKED ISSUE #52: document our image versioning strategy

### PR DESCRIPTION
**Pull request by charlesakalugwu**
closes #52

### CODE REVIEW

**Comment by jim-minter** [comment]
@charlesakalugwu please implement or close :)

File: docs/image_versioning_strategy.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```
+rhel7-<openshift_version>-%Y%m%d%H%M
+centos7-<openshift_version>-%Y%m%d%H%M
+```
+for example `rhel7-3.10-201810011519` or `rhel7-master-201810011519`
+
+
+### Container images
+
+All container images used by the , where possible, are tagged using the same version as on our enterprise build system
```
Comment (asalkeld) [inline docs/image_versioning_strategy.md]: used by the ???, ..

doesn't really make sense

File: docs/image_versioning_strategy.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+for example `rhel7-3.10-201810011519` or `rhel7-master-201810011519`
+
+
+### Container images
+
+Certain container images which are deployed as critical components of the OpenShift on Azure service are versioned, where possible, following the same scheme as official OpenShift releases
+```
+v<openshift_version>.<build_number>
+```
+for example `sync-image:v3.10.34`
```
Comment (mjudeikis) [inline docs/image_versioning_strategy.md]: we not quite do this now :/ can we have an issue for this in the main repo? :)

**Pull request merged**